pkg/controller/namespace/namespace_controller_utils.go

```diff
@@ -459,11 +460,18 @@ func ServerPreferredNamespacedGroupVersionResources(discoveryClient discovery.Di
 	if err != nil {
 		return results, err
 	}
+
+	allErrs := []error{}
 	for _, apiGroup := range serverGroupList.Groups {
-		preferredVersion := apiGroup.PreferredVersion
```

Comment:
if thirdpartyresources are the only type affected by this, and the version that would contain this workaround would also contain the fix on the API side to make the workaround no longer needed, I'd rather not add the workaround